packages/wallets/src/components/WalletsResetMT5Password/WalletSuccessResetMT5Password.tsx

```diff
@@ -19,9 +20,9 @@ const WalletSuccessResetMT5Password: FC<WalletSuccessResetMT5PasswordProps> = ({
 
     const renderButtons = useCallback(() => {
         return (
-            <WalletButton isFullWidth={isMobile} onClick={onClick} size='lg'>
+            <Button isFullWidth={isMobile} onClick={onClick} size='lg' textSize={isMobile ? 'md' : 'sm'}>
```

Comment:
better to change `isMobile` to !`isDesktop` for this

Reply:
Didn't change much to avoid conflicts with the tablet view card, but I'll update them to avoid confusion 👍